cmd/soroban-cli/src/commands/cfg/mod.rs

```diff
@@ -0,0 +1,24 @@
+mod migrate;
+
+use clap::Parser;
+
+/// Migrate config from previous versions.
```

Comment:
This comment looks like a description for the Migrate command. It needs to be on the `migrate::Cmd` struct in the other file.

Reply:
Actually, both ways work. We do it in both ways throughout the codebase. 😬